test/TestUtilities/Test.Utility/CommandRunner.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
nit: Look like you're removing a BOM mark.

Reply:
This modification was not by me, but happened automatically from VS.

Reply:
In general if you are not making any other changes to the file you should revert it.